.gitlab.mk

```diff
@@ -133,7 +133,7 @@ deploy:
 	echo ${GPG_SECRET_KEY} | base64 -d | gpg --batch --import || true
 	./tools/update_repo.sh -o=${OS} -d=${DIST} \
 		-b="${LIVE_REPO_S3_DIR}/${BUCKET}" build
-	if [ "${CI_COMMIT_TAG}" != "" ]; then \
+	if [[ ${GITHUB_REF} == refs/tags/* ]]; then \
```

Comment:
It'll not work on dash. `case` works with globs on bash and dash both:

```sh
$ case "${GITHUB_REF}" in refs/tags/*) echo ok ;; esac
ok
```

Reply:
Ok.